tasks/ast_tools/src/derives/content_hash.rs

```diff
@@ -13,13 +13,11 @@ define_derive! {
     pub struct DeriveContentHash;
 }
 
-const IGNORE_FIELDS: [(/* field name */ &str, /* field type */ &str); 6] = [
-    ("span", "Span"),
-    ("trailing_comma", "Span"),
-    ("this_span", "Span"),
-    ("scope_id", "ScopeId"),
-    ("symbol_id", "SymbolId"),
-    ("reference_id", "ReferenceId"),
+const IGNORE_FIELDS_TYPES: [/* type name */ &str; 4] = [
```

Comment:
This is 100% nit, but I think `IGNORE_FIELD_TYPES` (singular "field") would be more grammatical.

Reply:
Yes Sir.

Reply:
Thanks Rez.